apisix/utils/batch-processor.lua

```diff
@@ -200,10 +202,19 @@ function batch_processor:push(entry)
     end
     self.last_entry_t = now()
 
+    local entry_size = #entry
```

Comment:
The entry here is usually an object. Is there any real-world use case for this feature?

Reply:
Yes, sometimes we want to include request body or response body in the log.
Request body or response body may be large size.